src/autotrain/cli/run_setup.py

```diff
@@ -35,7 +35,7 @@ def __init__(self, update_torch: bool, colab: bool = False):
 
     def run(self):
         if self.colab:
-            cmd = "pip install -U xformers==0.0.22"
+            cmd = "pip install -U xformers==0.0.24"
```

Comment:
this file is specific to google colab. need to test if this version of xformers and torch work in colab :)